docs/source/common/loggers.rst

```diff
@@ -226,8 +231,8 @@ The :class:`~pytorch_lightning.loggers.WandbLogger` is available anywhere except
     class MyModule(LightningModule):
         def any_lightning_module_function_or_hook(self):
             some_img = fake_image()
-            self.logger.experiment.log({
-                 "generated_images": [wandb.Image(some_img, caption="...")]
+            self.log({
+                "generated_images": [wandb.Image(some_img, caption="...")]
```

Comment:
hey guys... we need to standardize for all loggers, not just wnb. let's sync up on this to make sure these changes aren't just for a single logger.

Reply:
Let me know if I need to update this on the other loggers as well.

I was just trying to take advantage of this PR to clean up the doc since I often get asked on when to use `self.log`, `self.logger.experiment.log` or even `self.logger[0].experiment.log` (I typically suggest to just try to use `self.log`).

Reply:
`self.log_images` could be the API used to log every image related artefacts.